code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -33,13 +33,19 @@
 		if(!fullpower)
 			src << "<span class='warning'>Your powers are useless on this holy ground.</span>"
 			return 0
+	if(check_holy())
+		if(!fullpower)
+			var/turf/simulated/T = get_turf(src)
+			if(T.lighting_lumcount > 2)
```

Comment:
Nah, just gotta do something like
```
#define VAMP_LIGHT_THRESHOLD 2
```
at the top of the file, and #undef it at the bottom